miso-web/src/main/webapp/scripts/list.js

```diff
@@ -258,6 +258,9 @@ ListUtils = (function() {
       }
     },
     render : {
+      archived : function(data, type, full) {
+        return data ? "🗄" : "";
```

Comment:
If the archived ones are hidden completely, could that be noted on the page somewhere?